### PR TITLE
fix(url): Broken link for meeting full stack app

### PR DESCRIPTION
**Pull request by MichaelWalker-git**
**Issue #:**
- Fix Broken link

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
- yes

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
- No

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
- No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by michhyun1 — APPROVED** [review]

**Comment by michhyun1** [comment]
need to run: npm run build:release

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ In addition to the below, here is a list of [all blog posts about the Amazon Chi
 
 ### High level
 
-- [High Level Architecture — Building a Meeting Application With the Amazon Chime SDK](https://aws.amazon.com/blogs/business-productivity/building-a-meeting-application-using-the-amazon-chime-sdk/)
+- [High Level Architecture — Building a Meeting Application With the Amazon Chime SDK](https://aws.amazon.com/blogs/business-productivity/build-meeting-features-into-your-amazon-chime-sdk-messaging-application/)
```
[the remark this answers is not in the document]
Comment (ltrung) [reply README.md:50]: PR #2824 suggested to use https://docs.aws.amazon.com/chime-sdk/latest/dg/service-architecture.html which I think is more correct.

[the remark this answers is not in the document]
Comment (MichaelWalker-git) [reply README.md]: I have updated the link with the high level arch documentation link.

**Comment by michhyun1** [comment]
Not sure whats changed but the doc sript should only have updated the readme docs - is there a reason there are that many doc changes?

**Comment by ltrung** [comment]
> Not sure whats changed but the doc sript should only have updated the readme docs - is there a reason there are that many doc changes?

It is because he created the PR from fork so typedoc just updated all the links to his fork :p

**Comment by ltrung** [comment]
Thanks for contributing. Merging as part of #3000 so closing this

**Pull request closed without merging**